README.md

```diff
@@ -137,8 +137,8 @@ To reliably check if an element is tabbable/focusable, Tabbable defaults to the
 
 The `displayCheck` configuration accepts the following options:
 
-- `full`: (default) Most reliably resembling browser behavior, this option checks that an element is displayed and all of his ancestors are displayed as well (notice that this doesn't exclude `visibility: hidden` or elements with zero size). This check is by far the slowest option as it will cause layout reflow.
-- `non-zero-area`: This option checks display under the assumption that elements that are not displayed have zero area (width AND height equals zero). While not keeping true to browser behavior, this option may be less intensive than the `full` option, and better for accessibility, as zero-size elements with focusable content are considered a strong accessibility anti-pattern.
+- `full`: (default) Most reliably resembling browser behavior, this option checks that an element is displayed and all of his ancestors are displayed as well (notice that this doesn't exclude `visibility: hidden` or elements with zero size).
```

Comment:
I think it's still worth pointing out that this option causes layout reflow, since that was a big point of contention in the past, which led to these 3 `displayCheck` options (where we only had the "full" in the past).

```suggestion
- `full`: (default) Most reliably resembling browser behavior, this option checks that an element is displayed and all of his ancestors are displayed as well (notice that this doesn't exclude `visibility: hidden` or elements with zero size). This option will cause layout reflow, however. If that is a concern, consider the `none` option.
```

Reply:
> I think it's still worth pointing out that this option causes layout reflow, since that was a big point of contention in the past

Absolutely!